sigs/fedora.md

```diff
@@ -0,0 +1,12 @@
+# Konflux + Fedora Special Interest Group
+
+## Purpose:
+
+* Operate and manage the [Fedora Konflux instance](https://gist.github.com/ralphbean/a3644111a549e8cedb0b207f90d42dc9).
+* Support people who are trying out experiments there.
+* Develop new ideas and PoCs for how Konflux might be used in Fedora.
+
+## Communication
+
+* Use the [#containers-sig](https://discussion.fedoraproject.org/tag/containers-sig) tag in Fedora Discussion for [async topics](https://discussion.fedoraproject.org/t/containers-sig-for-konflux-discussions/128061/12).
+* Every other week on Wednesday, we'll also hold an open sync call on Wednesday at 12:00 PM UTC, 8:00 AM US/Eastern at [this link](https://meet.google.com/wcr-uvhx-xug). Details are on the [shared calendar](https://calendar.google.com/calendar/u/0?cid=ZWViNjE1YTdjZTNmYWE0MmIyMjUwOGMwZWY4YTQ0NzcxM2FiNDYzNzg1YTRkMTljMGYyYmNjNTkyYjFkNjVlZEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
```

Comment:
Should we indicate that this is every other week starting October 16th, 2024?

Reply:
Well, let's leave out that detail. It'll be obvious if someone look at the calendar, but the start-date of the series will quickly become extra unnecessary information if merged in the markdown file itself.